Source/JavaScriptCore/wasm/WasmSlowPaths.cpp

```diff
@@ -647,6 +647,34 @@ WASM_SLOW_PATH_DECL(call_ref_no_tls)
     return doWasmCallRef(callFrame, instance, reference, instruction.m_typeIndex);
 }
 
+WASM_SLOW_PATH_DECL(tail_call)
+{
+    UNUSED_PARAM(callFrame);
+    auto instruction = pc->as<WasmTailCall>();
```

Comment:
This is not right, right? Do these just never get called?

Reply:
These are ccalls in the interpreter:
`callWasmCallSlowPath -> cCall4 -> JSC::LLInt:slow_path_wasm_tail_call.. (this) -> JSC::LLInt::doWasmCall`: encodes the callee's address and the instance, then returns to `slowPathForWasmTailCall` in the interpreter.